Add obsidian-editor-width-slider plugin

MugishoMp opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/MugishoMp/obsidian-editor-width-slider

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
Your manifest is missing the `authorUrl´ property, if you don't have a website, use your GH profile url.

[// Remember to rename these classes and interfaces!](https://github.com/MugishoMp/obsidian-editor-width-slider/blob/8adf887f3e0bd96c84836e595bbb68c7b5b1cff6/main.ts#L5)
Remove this comment

[// If the plugin hooks up any global DOM events (on parts of the app](https://github.com/MugishoMp/obsidian-editor-width-slider/blob/8adf887f3e0bd96c84836e595bbb68c7b5b1cff6/main.ts#L34-L45)
Remove this code, it's unused.

MugishoMp commented:
Thank you looking at it!

I have addressed all the issues that you indicated
- missing property added
- unnecessary comment removed
- unused code removed

MugishoMp commented:
@joethei I released version 1.0.2 which should have fixed all of the issues you addressed. Thank you for the feedback!
